docs/docs/index.md

```diff
@@ -155,16 +155,44 @@ run/debug configuration.
 
 ### VSCode
 
-Add following in the `settings.json`:
+Add the following to `.vscode/settings.json`:
 
 ```json
 {
-  ...
+  "python.pythonPath": "/usr/bin/python",
```

Comment:
Please remove this as it is very platform specific

Reply:
Ah, yes indeed, sorry! I wanted to emphasize that one can set the interpreter to the system one.